packages/next/README.md

```diff
@@ -15,13 +15,46 @@
   </a>
 </p>
 
+## The React Framework
+
+Next.js is used by the world's leading companies and independent developers. 
+
+* **Pre-Rendering** - Statically generated and server-rendered React applications have never been easier.
+* **Static Exporting** - No need to learn a new framework. Exporting a static site with Next.js is as easy as a single command.
+* **CSS-in-JS** - Next.js comes with `styled-jsx` included, but it also works with every CSS-in-JS solution you know and love.
+* **Built-in TypeScript support** - Next.js provides an integrated TypeScript experience out of the box, similar to an IDE.
```

Comment:
```suggestion
* **Built-in TypeScript Support** - Next.js provides an integrated TypeScript experience out of the box, similar to an IDE.
```

Reply:
Hi @Timer, @lfades suggested that this was redundant and the readme should directly link to https://nextjs.org/learn instead. Please let me know what you think, and if I should keep this on the Next.js readme.